upup/pkg/fi/cloudup/new_cluster.go

```diff
@@ -1055,7 +1066,21 @@ func setupNodes(opt *NewClusterOptions, cluster *api.Cluster, zoneToSubnetsMap m
 			g.Spec.NodeLabels["cloud.google.com/metadata-proxy-ready"] = "true"
 		}
 
-		g.Spec.MachineType = opt.NodeSize
+		for i, size := range opt.NodeSizes {
+			if i == 0 {
```

Comment:
same if condition twice?

Reply:
Thanks for catching. Was in a hurry earlier.